src/views/CoCurricularTranscript/index.tsx

```diff
@@ -32,6 +33,7 @@ const CoCurricularTranscript = () => {
           return;
         }
         setLoading(true);
+        setPreviousTitles([]);
```

Comment:
```suggestion
        setPreviousTitles([]);
```
make a comment about this bugging out BECAUSE of the default transcript item. or else this looks confusing